nextflow.config

```
@@ -1 +1,29 @@
+process {
+    executor='slurm'
+    queue="$SCXA_HPC_QUEUE"
+    // clusterOptions="$SCXA_HPC_OPTIONS"
+    time = '7 d'
+    memory = '4 GB'
+    queueSize=500
+    exitReadTimeout='100000 sec'
+    pollInterval = '5sec'
+    // error strategy
+    // errorStrategy = { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
+    // memory = { 4.GB * 2 ^task.attempt }
+    // maxRetries = 4
```

Hi @anilthanki , I'm not sure why this froze Nextflow.  Perhaps in Groovy (and thus Nextflow), ^ is actually the bitwise XOR operator, not the exponentiation operator

```suggestion
        // errorStrategy to handle processes killed due to memory issues
        errorStrategy = { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
        memory = { Math.min(4.GB * Math.pow(2, task.attempt), 256.GB) }
        maxRetries = 5
```